### ISSUE TITLE
grid checkbox label for assistive label / accessibility should be opt-in from grid

### ISSUE THREAD

**Issue by focbenz** [vaadin/framework#10561]
To improve grid performance it would be better to disable the accessibility text "Selects row number <x>." instead of just moving them 2000 pixels out of sight if you do not need the assitive text. Also copy and pasting areas from the grid would contain the otherwise not visible (English) assistive text.

*Background*
To have nice CSS styled checkboxes we use the CSS before/after pseudo elements that the Vaadin uses also for normal checkboxes in the valo theme.
The temp fix for the accessibility label needed for screenreaders introduced with #10223 in the Vaadin grid in V.8.1.7 / V.8.2.x also breaks custom styled checkboxes that are contained in the same label.

*Problem and Workaround*
Also the checkboxes are moved outside the view area and need to be corrected by adding back the missing 2000 pixels on x and y axis.

*Suggestion*
Add a server-side option to enable the accessibility option on the grid which might improve rendering performance.

*Additional remarks*
Also as knoobie mentioned in #10223 i18n is missing

**Comment by focbenz** [comment]
Do not stale this ... the issue was still being worked on and should be fixed.

**Comment by focbenz** [comment]
Thanks to knoobie for trying to provide this enhancement. 
As Vaadin 8 is no longer maintained actively I am closing this enhancement request.
